ui/src/components/CopyUrlButton.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { PrimaryButton, TerraTooltip } from "libs/common";
+import "@clr/icons/clr-icons.css";
+
+const style = {
+  copyUrlButton: {
+    marginRight: "16px"
+  }
+};
+
+export default class CopyUrlButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      buttonText: "Copy URL to clipboard"
+    };
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+  }
+
+  handleButtonClick() {
+    // Just change the button text for a second
+    this.setState({ buttonText: "Copied!" });
+    setTimeout(
+      () => this.setState({ buttonText: "Copy URL to clipboard" }),
+      1000
+    );
+  }
+
+  render() {
+    const { className } = this.props;
+
+    return (
+      <div onClick={this.handleButtonClick} style={style.copyUrlButton}>
```

Comment:
It's [recommended to use className rather than style](https://reactjs.org/docs/dom-elements.html#style).

In this repo we use the [material-ui styling solution](https://material-ui.com/customization/overrides/#1-specific-variation-for-a-one-time-situation).
- At the bottom of this file: `export default withStyles(styles)(CopyUrlButton);`
-
```
const { classes } = this.props;
... className={classes.copyUrlButton}
```

Reply:
Good to know, thanks! In this case, since we wanted to move the copyUrl button to the left we don't even need the extra styling. So I just removed this altogether.